src/stream/src/executor/backfill/arrangement_backfill.rs

```diff
@@ -540,7 +540,7 @@ where
                 .await?;
```

Comment:
nits: we can use `map` to call `KeyedRow::into_owned_row` to avoid introdcuing the `owned_row_iter` method.